README.md

```diff
@@ -59,11 +59,12 @@ The CloudFormation template for EKS Nodes is published in the S3 bucket
 `amazon-eks` under the path `cloudformation`. You can see a list of previous
 versions by running `aws s3 ls s3://amazon-eks/cloudformation/`.
 
-| CloudFormation Version | EKS AMI versions     | [amazon-vpc-cni-k8s](https://github.com/aws/amazon-vpc-cni-k8s/releases) |
-| ---------------------- | -------------------- | -------------------- |
-| 2018-08-21             | amazon-eks-node-v23+ | v1.1.0 |
-| 2018-08-30             | amazon-eks-node-v23+ | v1.1.0 |
-| 2018-11-07             | amazon-eks-node-v25+ | v1.2.1 (for t3 and r5 instances) |
+| CloudFormation Version | EKS AMI versions                      | [amazon-vpc-cni-k8s](https://github.com/aws/amazon-vpc-cni-k8s/releases) |
+| ---------------------- | ------------------------------------- | -------------------- |
+| 2018-08-21             | amazon-eks-node-v23+                  | v1.1.0 |
+| 2018-08-30             | amazon-eks-node-v23+                  | v1.1.0 |
+| 2018-11-07             | amazon-eks-node-v25+                  | v1.2.1 (for t3 and r5 instances) |
+| 2018-12-10             | amazon-eks-node-(1.11|1.10)-v20181210 | v1.2.1 |
```

Comment:
The pipe between 1.11 and 1.10 is causing this to not render correctly, can you change that to a comma?